src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -21,41 +25,74 @@
 import seedu.address.model.person.Name;
 import seedu.address.model.person.Person;
 import seedu.address.model.person.Phone;
+import seedu.address.model.person.RiskAppetite;
+import seedu.address.model.person.DisposableIncome;
+import seedu.address.model.Model;
```

Comment:
Should there be an import for ClientId?
```suggestion
import seedu.address.model.person.ClientId;
.
.
.
import seedu.address.model.person.Name;
import seedu.address.model.person.Person;
import seedu.address.model.person.Phone;
import seedu.address.model.person.RiskAppetite;
import seedu.address.model.person.DisposableIncome;
import seedu.address.model.Model;
```